src/client/app/containers/UIOptionsContainer.js

```diff
@@ -14,9 +14,24 @@ import { fetchMetersDataIfNeeded } from '../actions/meters';
  */
 function mapStateToProps(state) {
 	const sortedMeters = _.sortBy(_.values(state.meters.byMeterID).map(meter => ({ id: meter.id, name: meter.name.trim() })), 'name');
+
+	const timeInterval = state.graph.timeInterval;
+	const data = { datasets: [] };
```

Comment:
`data` seems to just be a wrapper around `datasets`. Why does it exist at all? Why not just use the `datasets` list as your variable?